build.gradle

```diff
@@ -1,30 +1,41 @@
-def modVersion = "3.0.29-GTNH"
+/*
```

Comment:
I guess this is an older buildscript, as it's missing version. This PR has already many changes, maybe a good idea would be to withhold and do it in new one.